models/r2p1d/model.py

```diff
@@ -114,10 +115,14 @@ def __iter__(self):
 
 class R2P1DLoader(RunnerModel):
   """Impl of loading video frames using NVVL, for the R(2+1)D model."""
-  def __init__(self, device):
+  def __init__(self, device,
+               num_clips_population=[1, 15], num_clips_weights=[10, 1]):
```

Comment:
Those arguments are never set outside; why don't we move the default values to where we call the constructor of  `R2P1DSampler` at [L121](https://github.com/snuspl/rnb/pull/71/files#diff-36b12e6629d5acd5fa5e770ebfc23b1bR121)?

Reply:
Same applies to where `R2P1DSampler()` is called

Reply:
Actually, the arguments can be set outside thanks to the changes introduced in #34.
Any entries in the configuration file that are not for `RESERVED_KEYWORDS` (in `benchmark.py`) will be propagted to the corresponding runners.
Also, I don't quite understand what you mean by 'move the default values to L121'.

Reply:
Sorry for the confusion. I meant to suggest setting default values of `R2P1DSampler`'s arguments consistently;  the argument `clip_length` also determines the number of clips in the sampler but its default value is exclusively specified in another place (L120). I thought that it'd be better to set those values in one place, and as per your considerate explanation, I think here (L118) would be a better place.
(i.e., `__init__(self, device, clip_length=8, num_clips_population=[1, 15], ...`).

Since it's a super-minor issue, we can skip this at this PR.